src/gemdat/plots/plotly/_shape.py

```diff
@@ -21,9 +24,8 @@ def shape(
     ----------
     shape : ShapeData
         Shape data to plot
-    bins : int | Sequence[float]
-        Number of bins or sequence of bin edges.
-        See [hist()][matplotlib.pyplot.hist] for more info.
+    nbins : int
+        Number of bins
```

Comment:
With this change the api is no longer consistent between the matplotlib version and the plotly version.

Reply:
True! I made them consistent using `bins`